benchmark/drivers.js

```diff
@@ -1,20 +1,20 @@
-'use strict';
+import NodeSqlite from 'sqlite3';
+import BetterSqlite from '../lib/index.js';
 
 /*
 	Every benchmark trial will be executed once for each SQLite3 driver listed
 	below. Each driver has a function to open a new database connection on a
 	given filename and a list of PRAGMA statements.
  */
 
-module.exports = new Map([
+export default new Map([
 	['better-sqlite3', async (filename, pragma) => {
-		const db = require('../.')(filename);
+		const db = BetterSqlite(filename);
 		for (const str of pragma) db.pragma(str);
 		return db;
 	}],
 	['node-sqlite3', async (filename, pragma) => {
-		const driver = require('sqlite3').Database;
-		const db = await (require('sqlite').open)({ filename, driver });
+		const db = await (await import('sqlite')).open({ filename, driver: NodeSqlite.Database });
```

Comment:
No, this is wrong in the benchmark. You are "preloading" the `better-sqlite3` but then importing `sqlite` in every iteration.